ASM/src/hacks.asm

```diff
@@ -2584,8 +2584,8 @@ skip_bombchu_bowling_prize_switch:
 .orga 0xEE7E4C
     jal Hit_Gossip_Stone
 
-.orga 0x26C10E3
-    .byte 0xFF ; Set generic grotto text ID to load from grotto ID
+;.orga 0x26C10E3
+;    .byte 0xFF ; Set generic grotto text ID to load from grotto ID
```

Comment:
Any reason this is commented out? If it's not necessary anymore, it'd be better to just remove it.

Reply:
Nope, missed this when I was cleaning up all the old stuff I kept for reference.